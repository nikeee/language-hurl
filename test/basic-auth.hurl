# Perform basic authentication with login `bob` and password `secret`.
GET https://example.org/protected
[BasicAuth]
bob: secret
