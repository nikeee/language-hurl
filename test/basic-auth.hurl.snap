># Perform basic authentication with login `bob` and password `secret`.
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hurl comment.line.number-sign.hurl
>GET https://example.org/protected
#^^^ source.hurl keyword.control.method.hurl
#   ^ source.hurl
#    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hurl string.unquoted.url.hurl string.unquoted.url.hurl
>[BasicAuth]
#^^^^^^^^^^^ source.hurl entity.name.section.hurl
>bob: secret
#^^^ source.hurl entity.name.tag.header.hurl entity.name.tag.header.hurl
#   ^ source.hurl entity.name.tag.header.hurl punctuation.separator.key-value.hurl
#    ^ source.hurl entity.name.tag.header.hurl
#     ^^^^^^ source.hurl entity.name.tag.header.hurl string.unquoted.hurl
>